auto_landing/include/auto_landing/detect_platform.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <ros/ros.h>
+#include <opencv2/opencv.hpp>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
+#include <image_transport/image_transport.h>
+#include <nav_msgs/Odometry.h>
+#include <geometry_msgs/Point.h>
+
+namespace ariitk::detect
+{
```

Comment:
Bracket should be placed after a space after `namespace ariitk::detect` . Follow the same convention below as well